src/aliceVision/hdr/sampling.cpp

```diff
@@ -169,8 +169,12 @@ bool Sampling::extractSamplesFromImages(std::vector<ImageSample>& out_samples, c
     {
         const float exposure = times[idBracket];
 
+        image::SImageReadOptions options;
+        options.outputColorSpace = image::EImageColorSpace::LINEAR;
```

Comment:
should use `colorspace` param

Reply:
oups !!! did it too early in the morning